libsolidity/formal/EncodingContext.h

```diff
@@ -139,7 +139,7 @@ class EncodingContext
 	void pushSolver();
 	void popSolver();
 	void addAssertion(smtutil::Expression const& _e);
-	size_t solverStackHeight() { return m_assertions.size(); } const
+	size_t solverStackHeight() const { return m_assertions.size(); }
```

Comment:
lol this is a funny one, the `const` in the current version actually carries over to the next function and makes the return value pointer const (`const smtutil::SolverInterface*`). In any case, I dont think this is used anywhere, either.